exercises/isbn-verifier/example.rs

```diff
@@ -0,0 +1,44 @@
+/// An ISBN type
+#[derive(PartialEq, Eq)]
+enum IsbnType {
+    Isbn10,
+    Isbn13,
+}
+
+/// Checks if an 'X' is valid at the given position for the given ISBN type
+#[allow(non_snake_case)]
+fn is_X_valid(position: &usize, isbn_type: &IsbnType) -> bool {
+    return (isbn_type == &IsbnType::Isbn10 && position == &9 ) ||
```

Comment:
optionally, you may omit the `return` here if you also omit the trailing semicolon. Up to you whether to do that.